src/cljs/harja/ui/grid/perus.cljs

```diff
@@ -1005,27 +1005,36 @@
        ")"))
 
 (defn arvo-ja-nappi
-  "Jos annettu ehto on true, näyttää gridissä napin, jolla kenttää voidaan muokata.
-   Jos ehto on false, piirtää kentän arvon tekstinä sekä napin kynä-ikonilla,
-   jolla arvoa voi muokata."
-  [{:keys [ehto-fn nappi-teksti nappi-optiot
-           uusi-fn muokkaa-fn arvo voi-muokata?
-           muokkaa-ikoni ikoninappi?] :as optiot}]
-  (if voi-muokata?
+  "Piirtää arvon ja napin, tai pelkän napin optioista riippuen.
+   Napilla voidaan muokata arvoa tai näyttää lisätietoa arvosta.
+
+  Optiot:
+  pelkka-nappi-fn               Jos ehto on false, piirtää kentän arvon tekstinä sekä napin.
+                                Jos palautta true, näyttää gridissä pelkän napin.
+                                Oletuksena tämä funktio palauttaa sisäisesti false.
+  pelkka-nappi-teksti           Teksti joka näytetään silloin kun piirretään pelkkä nappi.
+  pelkka-nappi-toiminto-fn      Funktio, jota kutsutaan silloin kun piirretään pelkkä nappi ja sitä klikataan.
+  nappi-optiot                  Lisäoptiot, jotka annetaan pelkälle napille tai napille arvon kanssa.
+  arvo-ja-nappi-toiminto-fn     Funktio, jota kutsutaan silloin kun piirretään arvo ja nappi, ja nappia painetaan.
+  arvo-ja-nappi-napin-teksti    Teksti tai ikoni, joka piirretään napille arvon kanssa.
+  ikoninappi?                   Jos true, arvon kanssa piirrettävä nappi käyttää ikoninappi-tyyliä"
+  [{:keys [pelkka-nappi-fn pelkka-nappi-teksti nappi-optiot
+           pelkka-nappi-toiminto-fn arvo-ja-nappi-toiminto-fn arvo
+           arvo-ja-nappi-napin-teksti ikoninappi?] :as optiot}]
+  (let [pelkka-nappi-fn (or pelkka-nappi-fn (constantly false))]
```

Comment:
Rajapinta muuttui hieman, lähinnä nimeämiset. Komponentti oletti aiemmin, että siinä näkyvällä napilla muokataan arvoa. Enää ei oleteta näin, vaan kyseessä on geneerinen arvo ja nappi.